### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maham360**
Updated README to include our specific rate_limit.

#669 #668

### CODE REVIEW

**Comment by kalbfled** [comment]
Where can I verify that 3000 is correct?  Is this set in AWS?

**Comment by maham360** [comment]
The `rate_limit` is `3000` under one of our service ID's and was checked in postman

File: app/README.md [hunk 1]
```diff
@@ -29,3 +29,6 @@ needs to be queried to get older stats. As a result, the `get_specific_template_
 `template/rest.py` relies on a method, `fetch_template_usage_for_service_with_given_template()`, that first
 queries the `ft_notification_status` table and then the `notifications` table if the user needs to get stats
 from today.
+
+## Definition of message_limit
+The service message_limit field defines the limit of the number of messages the user can send per day. In the POST request above, it is set to 100. The service called "BVA" was created in the staging environment to update the message_limit of all services from 1000 to 100. The service is called, the message limit is checked, and the service is sent. This service was created in Postman, and the daily message limit is required when creating a service, so now the default message_limit is 100.
```
Comment (jessecanderson) [inline app/README.md:34]: Do we really need all the details around `BVA` for message limits? Can we just list that the `message_limit` in the system was recently updated to default to 100? Or an example of creating a new service in Postman but setting that `message_limit` to 100?

File: app/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Rate Limiting for SMS Sender
 Users can set different rate limits and rate limit intervals for each SMS sender. Rate limit is defined as the number
-of messages that can be sent by the SMS sender within the rate limit interval.
+of messages that can be sent by the SMS sender within the rate limit interval.
```
Comment (kalbfled) [inline app/README.md:3]: Please remove the extra white space.

File: app/README.md [hunk 3]
```diff
@@ -29,3 +29,6 @@ needs to be queried to get older stats. As a result, the `get_specific_template_
 `template/rest.py` relies on a method, `fetch_template_usage_for_service_with_given_template()`, that first
 queries the `ft_notification_status` table and then the `notifications` table if the user needs to get stats
 from today.
+
+## message_limit default
+The service called "BVA" was created in the staging environment to update the message_limit of all services from 1000 to 100.
```
Comment (kalbfled) [inline app/README.md:34]: Is this specific to staging?

**Review by kalbfled — CHANGES_REQUESTED** [review]

File: app/README.md (hunk 3 above)
Comment (maham360) [reply app/README.md:34]: The BVA service was created in the Dev environment, message_limit applies to the service.

**Comment by k-macmillan** [comment]
The issues associated with this have been closed. Time to close this one.

**Pull request closed without merging**